doc/cpd/61.rst

```diff
@@ -0,0 +1,193 @@
+CPD-61: Host Abstraction
+========================
+
+.. todo::
+
+    WIP
+
+
+Metadata
+--------
+
+* CPD Version: `61 <https://github.com/projectatomic/commissaire/issues/61>`_
+* Status: Open
+
+Description
+-----------
+Today the ``Host`` abstraction is a simple model. It is saved and retrieved via
+persistent storage with the ``StorageService``. This CPD makes the ``Host`` abstraction
+into a model which may have it's data pulled from another system
+(EX: `CloudForms <https://www.redhat.com/en/technologies/management/cloudforms>`_).
+
+Rationale
+---------
+Most environments Commissaire will be used in will not be greenfield. They will
```

Comment:
Maybe define "greenfield" here, or link to a definition?  (https://en.wikipedia.org/wiki/Greenfield_project)